examples/notebook_tutorials/random_walk_tutorial.ipynb

```diff
@@ -0,0 +1,943 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quantum Walks With Cirq - Tutorial"
+   ]
+  },
```

Comment:
Add a blurb about what is about what this notebook about and also `To get started, we'll need to import these libraries`. Something like that.